src/audio/audio.cpp

```diff
@@ -144,7 +144,9 @@ void Audio::subscribeInput()
     if (!inputSubscriptions++)
     {
         openInput(Settings::getInstance().getInDev());
-        openOutput(Settings::getInstance().getOutDev());
+        if (!alOutDev)
+            openOutput(Settings::getInstance().getOutDev());
```

Comment:
Unfortunately I have a metric ton of things to do, so I can't afford to investigate audio code that is working fine in master.
Please find a way to test your code before sending PRs, I can't take in buggy untested code and I don't have the time to fix the bugs in every PRs.